src/js/themes/media-theme-micro.js

```diff
@@ -0,0 +1,538 @@
+      />
+    </svg>
+    <svg aria-hidden="true" viewBox="0 0 24 24" slot="exit">
+      <path
+        d="M20.25 14.5h-5a.76.76 0 0 0-.75.75v5a.75.75 0 1 0 1.5 0V16h4.25a.75.75 0 1 0 0-1.5Zm-5-5h5a.75.75 0 1 0 0-1.5H16V3.75a.75.75 0 1 0-1.5 0v5a.76.76 0 0 0 .75.75Zm-6.5 5h-5a.75.75 0 1 0 0 1.5H8v4.25a.75.75 0 1 0 1.5 0v-5a.76.76 0 0 0-.75-.75Zm0-11.5a.76.76 0 0 0-.75.75V8H3.75a.75.75 0 0 0 0 1.5h5a.76.76 0 0 0 .75-.75v-5A.76.76 0 0 0 8.75 3Z"
+      />
+    </svg>
+  </media-fullscreen-button>
+</template>
+
+<template partial="LiveButton">
+  <media-live-button
+    part="live seek-live button"
+    disabled="{{disabled}}"
+    aria-disabled="{{disabled}}"
+  >
+    <span slot="text">Live</span>
+  </media-live-button>
+</template>
+
+<template partial="TimeRange">
+  <media-time-range
+    part="time range"
+    disabled="{{disabled}}"
+    aria-disabled="{{disabled}}"
+  ></media-time-range>
+</template>
+
+<template partial="VolumeRange">
+  <span class="volume-range-span">
+    <media-volume-range
+      part="volume range"
+      disabled="{{disabled}}"
+      aria-disabled="{{disabled}}"
+    ></media-volume-range>
+  </span>
+</template>
+
+<media-controller style="--_control-bar-place-self:{{controlBarPlace ?? 'unset'}}">
+  <slot name="media" slot="media"></slot>
+
+  <template if="streamType == 'on-demand'">
+
+    <media-control-bar slot="centered-chrome">
+      <div class="control-group">
+        {{>PlayButton}}
+        {{>SeekBackwardButton}}
+        {{>SeekForwardButton}}
+        <span class="volume-group">
+          {{>MuteButton}}
+          {{>VolumeRange}}
+        </span>
+        {{>CaptionsButton}}
+        {{>AirplayButton}}
+        {{>CastButton}}
+        {{>PipButton}}
+        {{>FullscreenButton}}
+      </div>
+    </media-control-bar>
+    {{>TimeRange}}
+
+  </template>
+
+  <template if="streamType == 'live'">
+
+    <template if="targetLiveWindow == null">
+
+      <media-control-bar slot="centered-chrome">
+        {{>LiveButton}}
+        <div class="control-group">
+          <span class="volume-group">
+            {{>MuteButton}}
+            {{>VolumeRange}}
+          </span>
+          {{>CaptionsButton}}
+          {{>AirplayButton}}
+          {{>CastButton}}
+          {{>PipButton}}
+          {{>FullscreenButton}}
+        </div>
+      </media-control-bar>
+
+    </template>
+
+    <template if="targetLiveWindow > 0">
+
+      <media-control-bar slot="centered-chrome">
+        {{>LiveButton}}
+        <div class="control-group">
+          {{>PlayButton}}
+          {{>SeekBackwardButton}}
+          {{>SeekForwardButton}}
+          <span class="volume-group">
+            {{>MuteButton}}
+            {{>VolumeRange}}
+          </span>
+          {{>CaptionsButton}}
+          {{>AirplayButton}}
+          {{>CastButton}}
+          {{>PipButton}}
+          {{>FullscreenButton}}
+        </div>
+      </media-control-bar>
+      {{>TimeRange}}
+
+    </template>
+
+  </template>
+
```

Comment:
suggestion(non-blocking): We should strongly consider adding a slot for the mux player error dialog for feature parity.